.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.1.0
+  hooks:
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+  - id: check-yaml
+  - id: check-added-large-files
+- repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
+  rev: v2.2.0
+  hooks:
+  - id: pretty-format-yaml
+    args: [--autofix]
+- repo: https://github.com/pycqa/isort
+  rev: 5.10.1
+  hooks:
+  - id: isort
+    name: isort (python)
+    args: [--profile=black]
+    exclude: examples/.*.py
+- repo: https://github.com/psf/black
+  rev: 22.1.0
+  hooks:
+  - id: black
+    exclude: examples/.*.py
+- repo: https://github.com/Lucas-C/pre-commit-hooks-markup
+  rev: v1.0.1
+  hooks:
+  - id: rst-linter
```

Comment:
All checks in here need to be enforced by github actions and, ideally, tox to prevent commits by people using different tools from containing lots of unrelated formatting changes.  Do you know if rst-linter, pretty-format-yaml, and the four from the pre-commit org have standalone equivalents?

Reply:
> - All checks in here need to be enforced by github actions and,
> - ideally, tox to prevent commits by people using different tools from containing lots of unrelated formatting changes

for github action there is this https://github.com/pre-commit/action, which mean i need to create new github workflow yaml

second point : i just realize that you use tox to test the repo. i can convert those hooks to use tox using pre commit local hook https://pre-commit.com/#repository-local-hooks

>  Do you know if rst-linter, pretty-format-yaml, and the four from the pre-commit org have standalone equivalents?

can you explain more about this point?

but as note only rst-linter and language-formatters-pre-commit-hooks not pypi but both can be installed from git repo

four hooks from pre-commit are from pre commit itself